src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -44,6 +44,7 @@ object WorkflowManagerActor {
   case class WorkflowAbort(id: WorkflowId) extends WorkflowManagerActorMessage
   final case class WorkflowMetadata(id: WorkflowId) extends WorkflowManagerActorMessage
   final case class RestartWorkflows(workflows: Seq[WorkflowDescriptor]) extends WorkflowManagerActorMessage
+  final case class CallCaching(id: WorkflowId, parameters: Seq[(String, String)], call: Option[String]) extends WorkflowManagerActorMessage
```

Comment:
Currently I'm just passing the type through as the Spray API provides it.  Are you suggesting that should be wrapped in a case class or a `type`?